test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -3856,7 +3857,7 @@ TEST_F(AtenXlaTensorTest, TestRandperm) {
   torch::Tensor shuffle_cpu = CopyToDevice(shuffle, torch::kCPU);
   std::vector<xla::int64> shuffle_data(shuffle_cpu.data_ptr<int64_t>(),
                                        shuffle_cpu.data_ptr<int64_t>() + n);
-  EXPECT_TRUE(xla::IsPermutation(shuffle_data, n));
```

Comment:
can you make it EXPECT_TRUE(shuffle_data.size() == n && xla::IsPermutation(shuffle_data)); same for the other place. This way we don't lose the size check.